docs/reference/transform/apis/start-transform.asciidoc

```diff
@@ -61,6 +61,10 @@ Identifier for the {transform}.
 [[start-transform-query-parms]]
 == {api-query-parms-title}
 
+`start_after`::
+(Optional, string) Restricts the set of transformed documents to those with
+timestamps after this time. Only applicable for continuous transforms.
```

Comment:
It would be good to explicitly mention here that relative times like `now-30d` are supported.

Reply:
Done.